src/content/reference/manifest.mdx

```diff
@@ -717,6 +717,56 @@ volumes:
 
 Sets the working directory of your development container.
 
+### external (object, optional)
+
+A list of external resources to deploy with your development environment. This external resources contains the information for resources that are deployed outside the okteto cluster.
+
+```yaml
+external:
+  db:
+    icon: database
```

Comment:
Is `icon` already implemented? Or is the intention to prepare docs for upcoming implementation? If the latter, should we consider adding the icon details later on when the implementation is done & ready?

Reply:
Since only the current status will make it into 1.4 we have to remove icon field from the docs. I think it will be implemented next sprint